src/Agent/NewRelic/Profiler/MethodRewriter/Instrumentors.h

```diff
@@ -32,7 +32,11 @@ namespace NewRelic { namespace Profiler { namespace MethodRewriter
             auto instrumentationPoint = instrumentationSettings->GetInstrumentationConfiguration()->TryGetInstrumentationPoint(function);
             if (instrumentationPoint == nullptr)
             {
-                if (!function->ShouldTrace()) return false;
+                if (!function->ShouldTrace())
+                {
+                    LogTrace(L"No instrumentation point for ", function->ToString());
```

Comment:
concern: how much extra log traffic is generated when finest is enabled, and will it add value?

Reply:
Yeah that's definitely a concern. The ones you've called out add an extra 1-3 lines for any existing "possibly instrumenting" log line, which I think is manageable. I've noticed a recurring issue with customers/support where they're confused when they see "possibly instrumenting" but no follow-up, so I'm hoping this will help with that.

Reply:
Cool